apps/website/src/data/tech/network.ts

```diff
@@ -372,11 +372,50 @@ const data: RootNetworkItem[] = [
                 connection: { type: "ethernet", location: "wall" },
               },
               {
-                type: "camera",
-                name: "Chinchilla",
-                model: "AXIS M1065-L (Fixed)",
-                url: "https://www.axis.com/products/axis-m1065-l/",
-                connection: { type: "ethernet", location: "wall" },
+                type: "switch",
+                name: "Rat/Chin Switch",
+                model: "Ubiquiti USW-Lite-8-PoE",
+                url: "https://store.ui.com/us/en/pro/category/switching-utility/products/usw-lite-8-poe",
+                connection: { type: "ethernet", location: "buried" },
```

Comment:
🤔 The ethernet run to this switch is buried?

Reply:
Good catch, updated to wall.